### PR TITLE
Change SiaCoin to KiloSiaCoin

### PR DESCRIPTION
**Pull request by triazo**

### CODE REVIEW

File: site/scripts/ui/views/hosting.js [hunk 1]
```diff
@@ -19,8 +19,8 @@ ui._hosting = (function(){
             "conversion": 1/144
         },{
             "name": "Price",
-            "unit": "SC Per GB Per Month",
-            "conversion": 4/1000000000000
+            "unit": "KS Per GB Per Month",
+            "conversion": 4/1000000000000000
```
Comment (DavidVorick) [inline site/scripts/ui/views/hosting.js]: Can we break up the extraordinary amount of 0's into byte sized pieces:

4 / (1000 *1000 * 1000 * 1000 * 1000)

Comment (lukechampine) [inline site/scripts/ui/views/hosting.js]: or `1e12`

Comment (DavidVorick) [inline site/scripts/ui/views/hosting.js]: you can do that in javascript?

**Comment by DavidVorick** [comment]
There is a merge conflict, you'll have to fix that. Luke and I can walk you through the process the first few times, it can be tricky and easy to screw up.

You'll get the hang of it though.

**Comment by DavidVorick** [comment]
In general you don't want to work on your master branch. Hiccups and mistakes are easier to fix if you keep your own master synchronized with our master, and then you do all of your development on some branch that is not master.

**Pull request merged**